app/views/state_file/questions/shared/_review_header.html.erb

```diff
@@ -5,6 +5,24 @@
     </div>
     <div id="income-info" class="white-group">
       <div class="spacing-below-5">
+        <p><%= t(".income_forms_collected") %></p>
+        <ul class="list--bulleted">
+          <% if current_intake.state_file_w2s.present? %>
+            <li>W2</li>
+          <% end %>
+          <% if current_intake.state_file1099_rs.present? %>
+            <li>1099R</li>
+          <% end %>
+          <% if current_intake.state_file1099_gs.count > 0 %>
+            <li>1099G</li>
+          <% end %>
+          <% if current_intake.direct_file_json_data.interest_reports.count.positive? %>
+            <li>1099INT</li>
+          <% end %>
+          <% if current_intake.direct_file_data.fed_ssb.positive? || current_intake.direct_file_data.fed_taxable_ssb.positive? %>
+            <li>SSA1099</li>
```

Comment:
On the jira ticket, it says to use the dashed form names:

> W-2
> 1099-G
> 1099-R
> 1099-INT
> SSA-1099


> Reference: (however, use the dashed form names)

Reply:
ah, i was looking at the screenshot and missed the "however"